packages/@okta/vuepress-site/conductor.yml

```diff
@@ -2348,7 +2348,9 @@ redirects:
   - from: /docs/guides/oie-embedded-widget-use-cases/-/oie-embedded-widget-use-case-sign-in-soc-idp
     to: /docs/guides/oie-embedded-widget-use-case-sign-in-soc-idp/-/main/
   - from: /signup/oie.html
-    to: /signup/oie-preview.html
+    to: /signup/
+  - from: /signup/oie-preview.html
+    to: /signup/
```

Comment:
Need to change redirect on aws?

Reply:
I don't believe so, these routes under `signup/` are not specifically routed in AWS so they should still fall through to the vuepress site routing